pulsar-broker/src/test/java/org/apache/pulsar/broker/loadbalance/SimpleBrokerStartTest.java

```diff
@@ -97,4 +101,25 @@ public void testNoNICSpeed() throws Exception {
     }
 
 
+    @Test
+    public void testCGroupMetrics() throws IOException {
```

Comment:
We should check if the test really uses the `jdk.internal.platform.Metrics` not the old way?

Reply:
Good idea, we can assert `metrics != null`.